src/types/onyx/PolicyTag.ts

```diff
@@ -3,11 +3,17 @@ type PolicyTag = {
     name: string;
 
     /** Flag that determines if a tag is active and able to be selected */
-    enabled: boolean;
+    enabled?: boolean;
 
     /** "General Ledger code" that corresponds to this tag in an accounting system. Similar to an ID. */
     // eslint-disable-next-line @typescript-eslint/naming-convention
-    'GL Code': string;
+    'GL Code'?: string;
+
+    /** Nested tags */
```

Comment:
Huh? What are nested tags? 😕

Reply:
The comment naming is wrong , it is just a list of tags